.github/pull_request_template.md

```diff
@@ -1,21 +1,33 @@
 ## Contributor
 
-- [ ] Breaking changes enumerated in [the release issue](https://github.com/demergent-labs/azle/issues/2053)
-- [ ] New documentation enumerated in [the release issue](https://github.com/demergent-labs/azle/issues/2053)
 - [ ] Code has been declaratized
+- [ ] All new functions have JSDoc/Rustdoc comments
 - [ ] Error handling beautiful (no unwraps or expects etc)
 - [ ] Code tested thoroughly
-- [ ] All new functions have JSDoc/Rustdoc comments
+- [ ] PR title:
+    - [ ] Indicates breaking changes with suffix "(Breaking Changes)"
```

Comment:
What do you think about making the suffix `(breaking changes)`?

Reply:
I feel fine with that